docker/labels.go

```diff
@@ -11,10 +11,11 @@ type Label string
 
 // Libcompose default labels.
 const (
-	NAME    = Label("io.docker.compose.name")
-	PROJECT = Label("io.docker.compose.project")
-	SERVICE = Label("io.docker.compose.service")
-	HASH    = Label("io.docker.compose.config-hash")
+	NAME     = Label("io.docker.compose.name")
+	PROJECT  = Label("io.docker.compose.project")
+	SERVICE  = Label("io.docker.compose.service")
+	HASH     = Label("io.docker.compose.config-hash")
+	RECREATE = Label("io.docker.compose.recreate")
```

Comment:
@dnephin RECREATE is set by the user.  I don't think you really confirmed that we should remove the functionality, but I think the discussion has been lost by updates to the PR.  We can't just remove the ability for users to control the recreate behavior through labels.  I really like the feature, actually need the feature, but we will just do it in RancherOS code I guess.  

Regarding https://github.com/docker/libcompose/pull/89/files#r43012067 I don't want to change the approach of passing arguments in this PR.  This code is consistent with the rest of the code base so it doesn't make sense to do it differently.  If we think it should be done a different way then lets address that in a different PR.